mfr/extensions/tabular/render.py

```diff
@@ -41,7 +42,14 @@ def _render_grid(self, fp, ext, *args, **kwargs):  # assets_path, ext):
         """
         self._renderer_tabular_metrics = {}
 
-        sheets = self._populate_data(fp, ext)
+        try:
+            sheets = self._populate_data(fp, ext)
+        except _csv.Error as e:
```

Comment:
Is there a reason to handle this here instead of in the csv parsing library (`./libs/stdlib_tools.py`)?